src/client/components/MessageList/MessageList.js

```diff
@@ -2,16 +2,24 @@ import React from 'react';
 import Message from '../Message/Message';
 import PropTypes from 'prop-types';
 import './MessageList.css';
+import ProfileImage from '../ProfileImage/ProfileImage';
 
 // Loop through all the messages in the state and create a Message component
 const MessageList = ({ messages }) => {
-  return (
-    <div className="chat-messages__container">
-      {messages.map(({ index, username, message }) => (
-        <Message key={index} username={username} text={message} />
-      ))}
-    </div>
-  );
+  const messageComponent = messages.map(({ index, username, message }) => {
```

Comment:
Are you sure about this? It seems like the following is what you need: 
```suggestion
  const messageComponent = messages.map(({username, message }, index) => {
```

Reply:
Also, is it `username` or `userName`?